prebuild/prebuild.vcxproj

```diff
@@ -88,6 +88,7 @@
       <Optimization>Disabled</Optimization>
       <AdditionalIncludeDirectories>..\cppwinrt</AdditionalIncludeDirectories>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
+      <ControlFlowGuard>Guard</ControlFlowGuard>
```

Comment:
Build is failing on debug with this, conflicts with `/ZI`

Reply:
Thanks for catching. Fixed this and disabled CFG in the other Debug flavors.